### DISCUSSION TITLE
Can not compile sketch from Guislice builder

### DISCUSSION THREAD

**Discussion by gammacortex** [ImpulseAdventure/GUIslice#373] [Q&A]
Hi All, 

i have created a super simple project in GUIslice Builder. Just a "hello" text string on an empty background 
for an SSD1306 128 x 32 Adafruit OLED. 

When I try to compile the project I get :

arduino_GSLC.h:40:35: error: macro names must be identifiers
 #define                           "/"
                                   ^~~
exit status 1
macro names must be identifiers

What can be the problem ?

Thanks, Rob

**Comment by ImpulseAdventure** [comment]
Hi @Robertobit —

- Can you zip up & attach your Builder project folder to this discussion thread?
- What version of the Builder are you using?
- Within the Builder, were you specifying any custom fonts?
- What version of GUIslice do you have? (See /src/GUIslice_version.h)


thanks

**Reply by gammacortex** [reply]
Hi @ImpulseAdventure, 

the error came from the board definition in the board manager. I changed to the Arduino UNO, and it compiled without 
errors. Thank you. Problem is solved.
